src/assets/icons/vector_logo_tray.svg

```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 64 64" hola_ext_inject="disabled">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 32 32" hola_ext_inject="disabled">
```

Comment:
woops, my bad. actually tried something out, forgot to change em back, doesnt do anything major though.